change/@ni-nimble-components-b275af58-4f98-4b30-a09b-f91dc2833b9b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@munteannatan I notice this PR is still in a draft state and does not have any questions where you @mention anyone for feedback. To me that means this PR is still in development and you are not looking for feedback yet.

If you have specific questions you need answered before moving off Draft make sure they are asked in the draft with people mentioned.

If you are ready for review then make sure to switch from Draft.

If it's still in development, that's fine, just want to make sure you are not waiting on us for anything.

Reply:
sure thank you!